### PR TITLE
Added supported stages + video demo + started harness prerequisites.

### PR DESCRIPTION
**Pull request by SushrutHarness**
Updated spinnaker migrator docs. More updates to come.

### CODE REVIEW

File: docs/docs/spinnaker-migration/intro.md [hunk 1]
```diff
@@ -10,5 +12,24 @@ This tool was developed by Harness to assist in the migration process from Spinn
...
+| 2  | manualJudgment            | Approval          | Yes             |                                                        |
+| 3  | checkPreconditions        | Shell Script      | Yes             | Supports Expressions, and Multiple Check preconditions |
+| 4  | deleteManifest            | Delete            | Yes             |                                                        |
+| 5  | evaluateVariables         | JEXL Expression   | Yes             |                                                        |
+| 6  | wait                      | Wait              | Yes             |                                                        |
+| 7  | deployManifest            | Deploy            | Yes             |                                                        |
+| 8  | webhook                   | Custom Webhook    | Yes             |                                                        |
+| 9  | findImageFromTags         |                   | Yes             |                                                        |
+| 10 | bake                      | Bake Plugin       | Yes             | AMI Baking                                             |
```
Comment (edgarulg) [inline docs/docs/spinnaker-migration/intro.md:29]: Currently the bake in Spinnaker works for multi-cloud (AWS, GCP, etc). We only support AWS in the migrator so we should highlight it.

File: docs/docs/spinnaker-migration/intro.md [hunk 2]
```diff
@@ -10,5 +12,24 @@ This tool was developed by Harness to assist in the migration process from Spinn
...
+| -- | ------------------------- | ----------------- | --------------- | ------------------------------------------------------ |
+| 1  | pipeline                  | Pipeline Chaining | Yes             | Can only chain up to 2 pipelines                       |
+| 2  | manualJudgment            | Approval          | Yes             |                                                        |
+| 3  | checkPreconditions        | Shell Script      | Yes             | Supports Expressions, and Multiple Check preconditions |
+| 4  | deleteManifest            | Delete            | Yes             |                                                        |
+| 5  | evaluateVariables         | JEXL Expression   | Yes             |                                                        |
+| 6  | wait                      | Wait              | Yes             |                                                        |
+| 7  | deployManifest            | Deploy            | Yes             |                                                        |
+| 8  | webhook                   | Custom Webhook    | Yes             |                                                        |
```
Comment (ritek01) [inline docs/docs/spinnaker-migration/intro.md:27]: 8 | webhook | HTTP step | Yes
-- | -- | -- | --


Can you update this stage.

File: docs/docs/spinnaker-migration/intro.md [hunk 3]
```diff
@@ -10,5 +12,24 @@ This tool was developed by Harness to assist in the migration process from Spinn
...
+| 9  | findImageFromTags         |                   | Yes             |                                                        |
+| 10 | bake                      | Bake Plugin       | Yes             | AMI Baking                                             |
+| 11 | jenkins                   | Jenkins Step      | Yes             |                                                        |
+| 12 | findArtifactsFromResource | Shell Script      | Yes             |                                                        |
+| 13 | Find Image From Tags      | Plugin            | Yes             |                                                        |
+| 14 | AWS: invokeLambda         | Plugin            | Yes             |                                                        |
+| 15 | ShrinkCluster             | Plugin            | Yes             |                                                        |
+| 16 | ScaleDownCluster          | Plugin            | Yes             |
```
Comment (ritek01) [inline docs/docs/spinnaker-migration/intro.md:35]: there other supported stages 

You can have a check from : https://harness.atlassian.net/wiki/spaces/Armory/pages/21772730670/Migrator+Supported+Stages

17 | ResizeServerGroup | Plugin | Yes |  
-- | -- | -- | -- | --
18 | DisableCluster | Plugin | Yes |  
19 | ArtifactoryPromote | Http Step | Yes |  
20 | DestroyServerGroup | Plugin | Yes |  
21 | Deploy AMI Backed with Packer | Plugin | Yes

Comment (ritek01) [reply docs/docs/spinnaker-migration/intro.md:35]: Plugin [link](https://hub.docker.com/r/harnessdev/aws-drone-plugin) for stages : 

- ShrinkCluster
- ScaleDownCluster
- DisableCluster
- InvokeLambda
- ResizeServerGroup
- DestroyServerGroup

**Review by ritek01 — DISMISSED** [review]

File: docs/docs/spinnaker-migration/intro.md [hunk 4]
```diff
@@ -10,5 +12,29 @@ This tool was developed by Harness to assist in the migration process from Spinn
...
+| 2  | manualJudgment                | Approval          | Yes             |                                                        |
+| 3  | checkPreconditions            | Shell Script      | Yes             | Supports Expressions, and Multiple Check preconditions |
+| 4  | deleteManifest                | Delete            | Yes             |                                                        |
+| 5  | evaluateVariables             | JEXL Expression   | Yes             |                                                        |
+| 6  | wait                          | Wait              | Yes             |                                                        |
+| 7  | deployManifest                | Deploy            | Yes             |                                                        |
+| 8  | webhook                       | HTTP Step         | Yes             |                                                        |
+| 9  | findImageFromTags             |                   | Yes             |                                                        |
```
Comment (ritek01) [inline docs/docs/spinnaker-migration/intro.md:28]: 9 | findImageFromTags | Shell Script | Yes
-- | -- | -- | --


ref : https://harness.atlassian.net/wiki/spaces/Armory/pages/21772730670/Migrator+Supported+Stages

**Review by ritek01 — APPROVED** [review]

**Pull request merged**